lib/generator.js

```diff
@@ -294,57 +295,62 @@ function generateDoc(source, options) {
   else {
     tocGroups.sort(util.compareNatural);
   }
-  // Start markdown for TOC categories.
-  var tocMarkdown = ['<!-- div class="toc-container" -->\n'];
-  _.each(tocGroups, function(group) {
-    tocMarkdown.push(
-      '<!-- div -->\n',
-      '## `' + group + '`'
-    );
-
-    if (sortEntries && organized[group]) {
-      // Sort the TOC groups.
-      organized[group].sort(function(value, other) {
-        var valMember = value.getMembers(0),
-            othMember = other.getMembers(0);
-
-        return util.compareNatural(
-          (valMember ? (valMember + getSeparator(value)) : '') + value.getName(),
-          (othMember ? (othMember + getSeparator(other)) : '') + other.getName()
-        );
-      });
-    }
-    // Add TOC entries for each category.
-    _.each(organized[group], function(entry) {
-      var member = entry.getMembers(0) || '',
-          name = entry.getName(),
-          sep = getSeparator(entry),
-          title = escape((member ? (member + sep) : '') + name);
-
-      if (entry.isAlias()) {
-        // An alias has a more complex html structure.
-        var owner = entry.getOwner();
-        tocMarkdown.push(
-          '* <a href="#' + owner.getHash(style) + '" class="alias">`' +
-          title + '` -> `' + owner.getName() + '`' +
-          '</a>'
-        );
-      } else {
-        // Add a simple TOC entry.
-        tocMarkdown.push(
-          '* ' +
-          makeAnchor(
-            '#' + entry.getHash(style),
-            '`' + title + '`'
-          )
-        );
+
```

Comment:
Can you init `var tocMarkdown` outside of the if/else? That way it's easier to migrate to `let`

Reply:
Done

Reply:
Sorry about the force-push, was declared using 'let' (force of habit) but now uses `var`. I assume you're waiting until later to refactor declarations to let/const.